Defs/Ammo/HighCaliber/23x115mm.xml

```diff
@@ -0,0 +1,301 @@
+			<Ammo_23x115mm_HE>Bullet_23x115mm_HE</Ammo_23x115mm_HE>
+			<Ammo_23x115mm_Incendiary>Bullet_23x115mm_Incendiary</Ammo_23x115mm_Incendiary>
+			<Ammo_23x115mm_Sabot>Bullet_23x115mm_Sabot</Ammo_23x115mm_Sabot>
+		</ammoTypes>
+		<similarTo>AmmoSet_Autocannon</similarTo>
+	</CombatExtended.AmmoSetDef>
+
+	<!-- ==================== Ammo ========================== -->
+
+	<ThingDef Class="CombatExtended.AmmoDef" Name="Ammo23x115mmBase" ParentName="MediumAmmoBase" Abstract="True">
+		<description>Large caliber cartridge used by autocannons.</description>
+		<statBases>
+			<Mass>0.35</Mass>
+			<Bulk>0.332</Bulk>
+		</statBases>
+		<tradeTags>
+			<li>CE_AutoEnableTrade</li>
+			<li>CE_AutoEnableCrafting</li>
+		</tradeTags>
+		<thingCategories>
+			<li>Ammo23x115mm</li>
+		</thingCategories>
+		<stackLimit>1000</stackLimit>
+	</ThingDef>
+
+	<ThingDef Class="CombatExtended.AmmoDef" ParentName="Ammo23x115mmBase">
+		<defName>Ammo_23x115mm_AP</defName>
+		<label>23x115mm cartridge (AP)</label>
+		<graphicData>
+			<texPath>Things/Ammo/HighCaliber/AP</texPath>
+			<graphicClass>Graphic_StackCount</graphicClass>
+		</graphicData>
+		<ammoClass>ArmorPiercing</ammoClass>
+		<cookOffProjectile>Bullet_23x115mm_AP</cookOffProjectile>
+	</ThingDef>
+
+	<ThingDef Class="CombatExtended.AmmoDef" ParentName="Ammo23x115mmBase">
+		<defName>Ammo_23x115mm_Incendiary</defName>
+		<label>23x115mm cartridge (AP-I)</label>
+		<graphicData>
+			<texPath>Things/Ammo/HighCaliber/Incendiary</texPath>
+			<graphicClass>Graphic_StackCount</graphicClass>
+		</graphicData>
+		<ammoClass>IncendiaryAP</ammoClass>
+		<cookOffProjectile>Bullet_23x115mm_Incendiary</cookOffProjectile>
+	</ThingDef>
+
+	<ThingDef Class="CombatExtended.AmmoDef" ParentName="Ammo23x115mmBase">
+		<defName>Ammo_23x115mm_HE</defName>
+		<label>23x115mm cartridge (AP-HE)</label>
+		<graphicData>
+			<texPath>Things/Ammo/HighCaliber/HE</texPath>
+			<graphicClass>Graphic_StackCount</graphicClass>
+		</graphicData>
+		<ammoClass>ExplosiveAP</ammoClass>
+		<cookOffProjectile>Bullet_23x115mm_HE</cookOffProjectile>
+	</ThingDef>
+
+	<ThingDef Class="CombatExtended.AmmoDef" ParentName="Ammo23x115mmBase">
+		<defName>Ammo_23x115mm_Sabot</defName>
+		<label>23x115mm cartridge (Sabot)</label>
+		<graphicData>
+			<texPath>Things/Ammo/HighCaliber/Sabot</texPath>
+			<graphicClass>Graphic_StackCount</graphicClass>
+		</graphicData>
+		<statBases>
+			<Mass>0.257</Mass>
+		</statBases>
+		<ammoClass>Sabot</ammoClass>
+		<cookOffProjectile>Bullet_23x115mm_Sabot</cookOffProjectile>
+	</ThingDef>
+
+	<!-- ================== Projectile ================== -->
+
+	<ThingDef Name="Base23x115mmBullet" ParentName="BaseBulletCE" Abstract="true">
+		<graphicData>
+			<texPath>Things/Projectile/Bullet_Big</texPath>
+			<graphicClass>Graphic_Single</graphicClass>
+		</graphicData>
+		<projectile Class="CombatExtended.ProjectilePropertiesCE">
+			<damageDef>Bullet</damageDef>
+			<speed>141</speed>
```

Comment:
Sources I can find put the projectile velocity a bit lower than the sheet, closer to 700 - 715.
[Source 1.](https://www.arsenal-bg.com/c/artillery-rounds-32/23x115-mm-169)
[Source 2.](https://arconpartners.net/products/ammunition/medium-caliber/23-x-115-mm-round-with-high-explosive-incendiary-tracer-projectile-hei-t-for-23-mm-aircraft-automatic-guns-gsh-23-gsh-23l-and-gsh-6-23m/)

Reply:
The sources I've found state 2400 fps and or 715m/s +- 10m/s. as example: https://bulcomersks.com/military-products/ammunition/artilery-ammunitions/23x115mm-round-with-hei-projectile/

Reply:
I'm not seeing the 2400 fps (~730 m/s). Let's just roughly split the difference and go with 715 m/s.